README.md

```diff
@@ -1,17 +1,30 @@
 # rentify
 a service for landlords deployed as a collection of microservices running in docker containers
 
-### languages
-Java 8/9
-
-
-### frameworks
-Spring Boot 2.0
-Spring Cloud
-Netflix 0SS
-https://projectreactor.io/
-
-### other
-OAUTH 
-JWT
+## Technology Stack
+1. Languages (JVM based)
+  - Java (minimum v9)
```

Comment:
maybe 8?

Reply:
v8 will be at "end of life" in September (2018) and by using the classpath instead of the module path must of the frameworks in the ecosystem support v9+ (Hehehe some good developer-friendly features coming in v10)

Reply:
ok, good plan